doc_src/conf.py

```diff
@@ -18,226 +18,234 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-project_root = os.path.dirname(os.path.abspath('.'))
+project_root = os.path.dirname(os.path.abspath("."))
 sys.path.insert(0, project_root)
 
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
+# needs_sphinx = '1.0'
```

Comment:
This should be done in another PR.

Reply:
I mean, black changes single quotes to double quotes, but clearly not here. Black formatted this as if it's a regular comment, but in fact it is not.

Reply:
This is a comment, I would not expect it to do anything besides adding the extra space, I was referring to  _Cleaning this file out of autogenerated bloat_ btw :)

Reply:
That's the problem, it's not just a comment adding extra space, it's unused code. Black did nothing valuable for this piece of code, even worse, it introduced extra noise in the diff. So that's why the the proposal. Since you're touching this line of code anyway, you could remove it altogether. Same applies for other code generated by sphinx here.

Another option would be to format this comment according to black code style. But again, this is not just a comment, it's a commented-out code. Leaving such code is a shady practice to begin with.

Reply:
The diff size is irrelevant, what is important is that there is a single point in history where all the codebase has been formatted with black.